### PR TITLE
fix(chromium): wait for existing pages when connecting

### LINKED ISSUE #679: can't get browser's context pages after connect_over_cdp
**Test on Chrome Version 90.0.4430.212 (Official Build) (64-bit) (Windows 10 20H2 OS Build 19042.928)
chrome lunched with "--remote-debugging-port=9222"**

### code:
```
##########
from playwright.sync_api import sync_playwright
playwright = sync_playwright().start()
chrome = playwright.chromium.connect_over_cdp("http://localhost:9222")
c1 = chrome.contexts[0] # default context
print(c1.pages) # []
print(len(c1.pages)) # 0
##########
```

### but i am able to get all the opened pages after i open a new_page:
```
##########
from playwright.sync_api import sync_playwright
playwright = sync_playwright().start()
chrome = playwright.chromium.connect_over_cdp("http://localhost:9222")
c1 = chrome.contexts[0]
c1.new_page()
print(c1.pages)
print(len(c1.pages)) # 6 (5 opened pages + the new page)
##########
```

### PR DESCRIPTION
**Pull request by JoelEinbinder**
fixes https://github.com/microsoft/playwright-python/issues/679
﻿

### CODE REVIEW

File: src/server/chromium/crBrowser.ts [hunk 1]
```diff
@@ -61,12 +61,16 @@ export class CRBrowser extends Browser {
...
     }
     browser._defaultContext = new CRBrowserContext(browser, undefined, options.persistent);
-
     await Promise.all([
-      session.send('Target.setAutoAttach', { autoAttach: true, waitForDebuggerOnStart: true, flatten: true }),
+      session.send('Target.setAutoAttach', { autoAttach: true, waitForDebuggerOnStart: true, flatten: true }).then(async () => {
+        // Target.setAutoAttach has a bug where it does not wait for new Targets being attached.
+        // However making a dummy call afterwards fixes this.
+        // This can be removed after https://chromium-review.googlesource.com/c/chromium/src/+/2885888 lands in stable.
+        await session.send('Target.getTargetInfo');
```
Comment (yury-s) [inline src/server/chromium/crBrowser.ts:69]: We used to have code with `setDiscoverTargets` which did this, maybe it's easier to just revert that?

**Review by yury-s — APPROVED** [review]

File: src/server/chromium/crBrowser.ts (hunk 1 above)
Comment (JoelEinbinder) [reply src/server/chromium/crBrowser.ts:69]: The code with setDiscoverTargets was way more complicated had bugs in it related to connecting twice to service workers.

**Comment by JoelEinbinder** [comment]
Looks like this is causing an electron test to fail. Looking into it

**Comment by JoelEinbinder** [comment]
Electron had a race where it was not expecting pages to appear before the browser was finished launching. I fixed it, and a test that was relying on that behavior. This could affect user code. I don't think we have a problem, because we used `firstWindow` everywhere in the docs. @pavelfeldman what do you think?

**Pull request merged**